src/peers/WorldMap/WorldMap.js

```diff
@@ -4,23 +4,16 @@ import { connect } from 'redux-bundler-react'
 import { AutoSizer } from 'react-virtualized'
 import { translate } from 'react-i18next'
 import * as d3 from 'd3'
-import * as topojson from 'topojson'
-
-import worldData from './world.json'
+import staticMapSrc from './StaticMap.svg'
 
 const WorldMap = ({ t }) => {
   return (
-    <div className='flex w-100 mb4' style={{ 'height': '550px' }}>
-      <AutoSizer>
-        { ({ height, width }) => (
-          <GeoPath width={width} height={height}>
+    <div className='flex w-100 mb4' style={{ 'height': '550px', background: `transparent url(${staticMapSrc}) center` }}>
```

Comment:
Adding `no-repeat` fixes the https://github.com/ipfs-shipyard/ipfs-webui/pull/808#issuecomment-424281275 issue:

```js
style={{ 'height': '550px', background: `transparent url(${staticMapSrc}) center no-repeat` }}
```